bgpd/bgp_mplsvpn.c

```diff
@@ -502,6 +502,7 @@ leak_update(struct bgp *bgp, /* destination bgp instance */
 	struct bgp_path_info *bpi;
 	struct bgp_path_info *bpi_ultimate;
 	struct bgp_path_info *new;
+	char buf_prefix[PREFIX_STRLEN];
```

Comment:
Can't find where is this used (populated)?

Reply:
Yes, we will remove it.
As per previous version of the code, we did log the prefix.
Now, we do not this.

Reply:
Taken care